src/generators/default/date/date.test.ts

```diff
@@ -6,12 +6,21 @@ import { DateGenerator } from '.';
 describe('create dates', () => {
 	const core = new Core().register([DateGenerator]);
 
+	test.fails('produces a valid date', () => {
```

Comment:
We'll need to look into this one.
Probably give it a fixed min and max to create a valid date.